tools/docker/docker-compose-mac.yml

```diff
@@ -165,5 +171,26 @@ services:
       retries: 3
       start_period: 5s
 
+  eda-webhook-api:
```

Comment:
Without modify the nginx conf (aside of the url_prefix options) , this new service would be just a resource consumer. Anyway, if you want to add it, docker-compose-dev should have it too.